pkgs/tools/misc/zsjed/default.nix

```diff
@@ -0,0 +1,68 @@
+{ coreutils
+, diffutils
+, fetchurl
+, findutils
+, gawk
+, lib
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zsjed";
+  version = "0.3";
+
+  src = fetchurl {
+    url = "https://ytrizja.de/distfiles/zsjed-${version}.tar.gz";
+    sha256 = "1rcrvrqjsv80ldz7cscqbrf98jb0yd2in97irrzlpask699z5k29";
+  };
+
+  nativeBuildInputs = [
+    diffutils
+  ];
+
+  buildInputs = [
+    coreutils # sort
+    findutils
+    gawk
+  ];
+
+  doCheck = true;
+  enableParallelBuilding = true;
+
+  # doesn't test zsjed-mkls
+  checkPhase = ''
+    mkdir tmp
+    cd tmp
+    touch f1 f2
+    echo f1 > l
+    echo f2 >> l
+    ../zsjed-clear l
+    echo "a b c d e f g h i j k" > i
+    ../zsjed-spread i l
+    ../zsjed-collect j l
+    cmp i j
+    cd ..
+    rm -rf tmp
+  '';
+
+  # the Makefile does not work for this, as it assumes the wrong prefix
+  installPhase = ''
+    install -D -t $out/bin zsjed-collect zsjed-clear zsjed-spread zsjed-mkls
+  '';
+
+  postFixup = ''
+    sed -i "2 i export PATH=${lib.makeBinPath buildInputs}:\$PATH" $out/bin/zsjed-mkls
```

Comment:
This is suboptimal, but I'm unsure how to use the `substitute*` fn's to put the appropriate paths in there...

Reply:
I think you might want to use makeWrapper here. You can take a look at somafm-cli for a simple example.